froide/foirequest/views/make_request.py

```diff
@@ -133,6 +133,9 @@ def get_js_context(self):
             "settings": {
                 "user_can_hide_web": settings.FROIDE_CONFIG.get("user_can_hide_web"),
                 "user_can_create_batch": self.can_create_batch(),
+                "non_meaningful_subject_regex": settings.FROIDE_CONFIG.get(
```

Comment:
This should have a default value of `[]` so the JS `.map` doesn't crash on `null` value.

Reply:
✅